### ISSUE TITLE
Citizens rendered useless for me.

### ISSUE THREAD

**Issue by almind1** [CitizensDev/Citizens#760]
whenever i type /npc the only thing that comes up is "type /npc help".  When i type any command to do with citizens 1.1.5, it says the same. help?

**Comment by JaredTS486** [comment]
Yes its saying you need to type /npc help. That will give you all of the neccesary commands. There are also:
/citizens help
/npc help
/guard help
/wizard help
/trader help
/blacksmith help
etc...

**Comment by fullwall** [comment]
There should be an error in server log. Could you post that please?
